code/game/atoms.dm

```diff
@@ -1399,3 +1399,9 @@
 /atom/proc/InitializeAIController()
 	if(ai_controller)
 		ai_controller = new ai_controller(src)
+
+//Update the screentip to reflect what we're hovering over
+/atom/MouseEntered(location, control, params)
+	. = ..()
+	// Statusbar
+	status_bar_set_text(usr, name)
```

Comment:
On high pop this is one of the most overtiming functions (Even when doing nothing!!)
Just simply adding /atom/MouseEntered and doing NOTHING with the result is bad. This is why we made an effort to remove it entirely.

Plus its a TG port so I automatically don't like it.